features/search/events.feature

```diff
@@ -1,12 +1,25 @@
 Feature:
-
-  Scenario: The search city is base on user's city
+  Background:
     Given the following fixtures are loaded:
       | LoadAdherentData |
+      | LoadEventData    |
+
+  Scenario: The search city is base on user's city
```

Comment:
The search city is base**d** on user's city

Reply:
fixed